hanging_threads.py

```diff
@@ -19,13 +19,6 @@
 """
 import sys
 import threading
-try:
-    try:
-        from threading import _get_ident as get_ident
-    except ImportError:
-        from threading import get_ident
-except ImportError:
-    from thread import get_ident
```

Comment:
looks like we do not need this because we use threading. current_thread, right? :)